pywbem_mock/_wbemconnection_mock.py

```diff
@@ -546,20 +845,20 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         mofcomp.compile_string(mof_str, namespace)
 
-    def compile_schema_classes(self, class_names, schema_pragma_file,
+    def compile_schema_classes(self, class_names, schema_pragma_files,
                                namespace=None, verbose=False):
         # pylint: disable=line-too-long
         """
-        Compile the classes defined by `class_names`  and all of their dependences.
-        The class names must be classes in the define schema and with
-        pragma statements in `schema_pragma_file`.
-        `schema_pragma_file` must be in a directory that also encompasses
-        the MOF files for all of the classes defined in the
-        `schema_pragma_file` and the dependencies of those classes. While
-        the relative paths of all of the CIM class files is defined in the
-        `schema_pragma_file` the pywbem MOF compiler may also search for
-        dependencies (ex. superclasses, references, etc.) that are not
-        specifically listed in the `class_names` and the path of the
+        Compile the classes defined by `class_names`  and all of their
+        dependences. The class names must be classes in the defined schema and
+        with pragma statements in a `schema_pragma_file`. Each
```

Comment:
This occurrence of `schema_pragma_file` and the following ones in this paragraph are all about the file itself, and not about the parameter. Therefore, change them to "schema pragma file".

Reply:
DONE